src/library/Utilities/RemoteContainerConnectionDetailsResolver.cs

```diff
@@ -210,25 +210,29 @@ internal class RemoteContainerConnectionDetailsResolver
             CancellationToken cancellationToken)
         {
             (var owningSetName, var owningObjectType) = GetOwningSetFromPods(pods, remoteContainerConnectionDetails.ServiceName);
-            _log.Verbose($"Owning object type: {owningObjectType}");
+            _log.Info($"Owning object type: {owningObjectType}");
             if (string.Equals(owningObjectType, "ReplicaSet"))
             {
+                _log.Info("Inside if condition for replicaset");
                 // There is a single common replicaSet owning all the pods. Time to make sure there is a deployment owning it
                 var replicaSet = await _kubernetesClient.GetV1ReplicaSetAsync(remoteContainerConnectionDetails.NamespaceName, owningSetName, cancellationToken: cancellationToken);
                 if (replicaSet == null || replicaSet.Metadata?.OwnerReferences == null)
                 {
                     // replicaSet not found or no deployment owning it
+                    _log.Info("replicaset is null:"+ replicaSet);

```

Comment:
💡 nit: most of the other logging has capital initial letter - just fyi - in case this is anything you want to cater.

Reply:
these are temporary logs once E2E and user confirms it is working I will remove them. Thank you.

Reply:
Testing team confirmed it is working.